### ISSUE TITLE
Keyboard observer moves all content up above keyboard

### ISSUE THREAD

**Issue by NicholasBellucci** [AndreaMiotto/PartialSheet#35]
It appears that the new addition of keyboard observers causes elements like a tab bar to shift above the keyboard in instances where the partial sheet is added to the TabView and a keyboard is presented at any time, sheet presented or not. I think I have a solution that works. 

Instead of `.padding(.bottom, self.offset)` in the PartialSheetViewModifier on line 79, the negative offset should be added to `iPhoneSheet()` as such `.offset(x: 0, y: -self.offset)` on like 113.

```swift
if deviceType == .iphone {
    iPhoneSheet()
        .offset(x: 0, y: -self.offset)
        .edgesIgnoringSafeArea(.vertical)
}
```

From my testing this solution will prevent the whole content from receiving a padding when the keyboard presents.

**Comment by Danny-SS** [comment]
Thanks, Nicholas.  This saved me! 

 In areas of my app not even using any partial sheets, every time I was tapping into a TextField, I was seeing my tab bar above the keyboard.  I did not realize it was this newly installed package and thought it was a KeyboardAdaptive modifier I use for normal SwiftUI Views for handling keyboard notifications and adjustments.

I first commented out the keyboard observers in PartialSheetViewModifier (.onAppear at line 80).  My app was back to normal.  Searched here and found your solution.  So far it is working.

**Comment by AndreaMiotto** [comment]
Hi @NicholasBellucci, I cannot reproduce your problem, can you share an image? 
Would you like to open a PR with the fixed part?

Thank you

**Comment by AndreaMiotto** [comment]
@NicholasBellucci I'm trying to resolve this issue, can you paste here you solution?

**Comment by AndreaMiotto** [comment]
Should be fixed, già a try with v2.1.6
